### PR TITLE
Update features.rst

### PR DESCRIPTION
**Pull request by ghost**
chore(docs): improved feature listing in intro docs

### CODE REVIEW

**Comment by hypeJunction** [comment]
Can you just include the text you want to appear in the final copy? That way we can just merge it and you get a mention in the release notes ;)

File: docs/intro/features.rst [hunk 1]
```diff
@@ -29,12 +30,12 @@ For admins
  * Responsive, mobile-friendly design
  * RSS support
  * Activity stream
- * Plugins for common content types like blogs, bookmarks, files, microblogging, private messages, documents, message boards, discussion 
+ * [suggestion: "Bundled"] Plugins for common content types like blogs, bookmarks, files, microblogging, private messages, documents, message boards, discussion
```
Comment (hypeJunction) [inline docs/intro/features.rst]: Just say: Bundled plugins ...

**Comment by michelesbw** [comment]
Ha ha no need mentions of course, my pleasure contributing in someway.
Ok will improve on this as well :)

**Comment by michelesbw** [comment]
But is it ok to make PRs like this?

**Comment by hypeJunction** [comment]
Sure, if you think it improves comprehension of docs.

**Comment by jdalsem** [comment]
Only issue is with the commit message

**Comment by michelesbw** [comment]
Sorry Jeroen what do you mean?

**Comment by jdalsem** [comment]
http://learn.elgg.org/en/latest/contribute/code.html#commit-message-format

in your case it would be 
chore(docs): improved feature listing in intro docs

**Comment by michelesbw** [comment]
Ha ha! I never thought I could become a contributor so automatically missed the whole guide there....thanks, will read more

**Comment by ewinslow** [comment]
@michelesbw thanks for the contribution! 1.10 is a bit old now. I think we'd want this to go on the 1.11 branch. And there is a bunch of cleanup to do regarding the commit message and contents of the docs. Hopefully you'll have time to read up on how to fix those issues and then re-submit this PR against 1.11!

**Pull request closed without merging**